docs/amber.md

```diff
@@ -0,0 +1,138 @@
+# Amber
+
+Amber is a multi-API shader test framework.Graphics bugs can be captured and
+communicated through a scripting interface. This removes the need to program
+to the low level interface when reproducing bugs.
+
+Amber is broken into multiple parts, the applications, the parsing components
+and the script execution.
+
+## Applications
+There are currently two applications, the `samples/amber.cc` amber application
+and the amber integration into the [CTS](https://github.com/KhronosGroup/VK-GL-CTS/tree/master/external/vulkancts/modules/vulkan/amber). These applications are responsible
```

Comment:
CTS --> "Vulkan Conformance Test Suite"

Reply:
done. Made it "Vulkan Conformance Test Suite 'CTS'" as I use CTS later.